### ISSUE TITLE
About using weak-labels

### ISSUE THREAD

**Issue by JoyHuYY1412** [Jia-Research-Lab/Context-Aware-Consistency#26]
Hi, thank you for your great work! 

Could you please give me some advice on using weak labels? Are the classification model are first pre-trained on ImageNet are next fine-tuned on specific weak-annotated datasets like CityScapes?

**Comment by X-Lai** [comment]
Thanks for your interest in our work. For the part of using weak labels, I recommend you to take a look at [CCT](https://github.com/yassouali/CCT). We follow its implementation to conduct the experiments.
